app/frontend/tsconfig.json

```diff
@@ -14,7 +18,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react-jsx"
```

Comment:
This was automatically picked up by create react app since we recently bumped TypeScript to 4.1.x, which supports the new JSX transform. That means unless you need hooks from React, we don't need `import React from 'react'` anymore

I thought I'll leave it in since it's likely a no-brainer improvement to the production build